website/content/docs/nia/configuration.mdx

```diff
@@ -147,7 +147,7 @@ task {
 - `name` - (string: required) Name is the unique name of the task (required). A task name must start with a letter or underscore and may contain only letters, digits, underscores, and dashes.
 - `enabled` - (bool: true) Enable or disable a task from running and managing resources.
 - `providers` - (list[string]) Providers is the list of provider names the task is dependent on. This is used to map [Terraform provider configuration](#terraform-provider) to the task.
-- `services` - (list[string]) An optional list of logical service names or service IDs that the task monitors in Consul Catalog for changes. `services` can be configured with a `condition` block of type `consul-kv`, `catalog-services`, or `schedule` to provide additional services source input to the task. Or it can be configured with no `condition` block to set the task's condition so that the task triggers when the services change. Any service value not explicitly defined by a `service` block with a matching ID is assumed to be a logical service name in the default namespace.
+- `services` - (list[string]) An optional list of logical service names or service IDs that the task monitors in Consul Catalog for changes. `services` can be configured with a `condition` block of type `consul-kv`, `catalog-services`, or `schedule` to provide additional services source input to the task. Alternatively, `services` can also be configured with without a `condition` block. Using `services` without a condition block results in a task triggering when the services change. Any service value not explicitly defined by a `service` block with a matching ID, is assumed to be a logical service name in the default namespace.
```

Comment:
I added some clarification in the comment above for what I was hoping to get across. To combine with your suggestions, what are your thoughts on:

```
- `services` - (list[string]) Specifies an optional list of logical service names or service IDs that the task monitors for changes in the Consul catalog. The `services` can act in different ways depending on the configuration of the task's `condition` block:
  - no `condition` block: `services` will act as the task's condition and provide the services information as source input
  - `condition` block configured for type `services`: `services` is incompatible with this type of `condition` because both configure the services source input. CTS will return an error.
  - `condition` block configured for all other types: `services` will act only to provide services source input.

  Service values that are not explicitly defined by a `service` block that have a matching ID are assumed to be logical service names in the `default` namespace.
```